.gitignore

```diff
@@ -20,3 +20,9 @@
 
 # Go workspace file
 go.work
+
+# env file
+.env
+
+# Autogenerated files from goplus generate
+gop_autogen.go
```

Comment:
If I understand correctly, the `.go` files generated by `gop` are welcome to be put into repo

Reply:
This file has caused many conflicts before.

I have removed it.